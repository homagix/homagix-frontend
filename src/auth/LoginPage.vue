<script setup lang="ts">
import { useRouter } from "vue-router"
import AppButton from "../components/AppButton.vue"

const router = useRouter()
</script>

<template>
  <h2 class="title is-4">Willkommen!</h2>
  <p>
    Du bist bereits registriert? Um deine Daten zu sch√ºtzen, musst du den Bookmark verwenden, den du nach der
    Registrierung erhalten hast.
  </p>

  <p>Du willst dich neu registrieren? Dann geht's <router-link to="/register">hier entlang</router-link></p>

  <div class="button-list">
    <AppButton @click="router.back">Abbrechen</AppButton>
  </div>
</template>

<style lang="scss">
.button-list {
  margin-top: 1rem;
}
</style>
