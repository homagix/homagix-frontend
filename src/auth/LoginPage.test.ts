import { createStore } from "vuex"
import { describe, expect, it } from "vitest"
import { mount } from "@vue/test-utils"
import LoginPage from "@/auth/LoginPage.vue"
import Router from "@/router"

const router = Router(createStore({}))

describe("LoginPage", () => {
  it("should render", () => {
    const wrapper = mount(LoginPage, { global: { plugins: [router] } })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
