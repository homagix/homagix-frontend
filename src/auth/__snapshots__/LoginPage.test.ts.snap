// Vitest Snapshot v1

exports[`LoginPage > should render 1`] = `
"<h2 class=\\"title is-4\\">Willkommen!</h2>
<p> Du bist bereits registriert? Um deine Daten zu schÃ¼tzen, musst du den Bookmark verwenden, den du nach der Registrierung erhalten hast. </p>
<p>Du willst dich neu registrieren? Dann geht's <a href=\\"/register\\" class=\\"\\">hier entlang</a></p>
<div class=\\"button-list\\"><button data-v-41d81733=\\"\\">Abbrechen</button></div>"
`;
